libavcodec/vaapi_av1.c

```diff
@@ -63,6 +63,9 @@ static int vaapi_av1_start_frame(AVCodecContext *avctx,
     int err = 0;
     int apply_grain = !(avctx->export_side_data & AV_CODEC_EXPORT_DATA_FILM_GRAIN) && film_grain->apply_grain;
     uint8_t remap_lr_type[4] = {AV1_RESTORE_NONE, AV1_RESTORE_SWITCHABLE, AV1_RESTORE_WIENER, AV1_RESTORE_SGRPROJ};
+    uint8_t segmentation_feature_signed[AV1_SEG_LVL_MAX] = {1, 1, 1, 1, 1, 0, 0, 0};
```

Comment:
this can be split to multiple commits
each - is a commit

Reply:
OK, I will check it. thanks.